argus/backend/controller/api.py

```diff
@@ -386,13 +387,21 @@ def test_info():
 @api_login_required
 def test_results():
     test_id = request.args.get("testId")
+    start_date_str = request.args.get("startDate")
+    end_date_str = request.args.get("endDate")
+
     if not test_id:
         raise Exception("No testId provided")
+
+    start_date = datetime.fromisoformat(start_date_str).astimezone(timezone.utc) if start_date_str else None
```

Comment:
What does the input string here look like? My concern is potential shift of the requested data +/- offset from UTC if input string doesn't have `Z` at the end or timezone specified.

Reply:
input string is just the date e.g. 2024-10-30